Add Access Control (#10166)
marmelab/react-admin: docs/List.md

```

`useListController` returns callbacks to sort, filter, and paginate the list, so you can build a complete List page. Check [the `useListController`hook documentation](./useListController.md) for details.

> ## Controlling Access To The List
> 
> ### Controlling Access In Exports

Reviewer: ```suggestion
## Access Control

### Exported Fields
```

> Should your authProvider implements the [`canAccess` method](./AuthProviderWriting.md#canaccess), the [`exporter`](#exporter) will call it for each record property to ensure it only exports fields to which users have access to. For instance, given the following record shape:

Reviewer: ```suggestion
Should your authProvider implement the [`canAccess` method](./AuthProviderWriting.md#canaccess), the [`exporter`](#exporter) will call it to export only fields to which the current user has access to. For instance, given the following record shape:
```

```json
{
    "id": 0,
    "title": "Never Gonna Give You Up",
    "author": "Rick Astley",
}
```

The `authProvider.canAccess` method will be called 3 times with the following parameters:

> 1. `{ action: "read", resource: "posts.id" }`

Reviewer: I think we should use the `export` action for that. This allows to have a distinction between fields that are shown and the ones that are exported.
